### ISSUE TITLE
Specify which files to download based on filename pattern

### ISSUE THREAD

**Issue by XioNoX** [mauriciojost/olympus-photosync#33]
I upgraded photosync from 0.6 to 0.9 and the regex I've been using to only copy ORF files is not working anymore.

I tried to update it using the new default one without success.

What I used to use was:
`photosync -r ".*=.*,(\\w+\\.ORF),(\\d+),.*" -o .`

And some examples of what I tried:
```
photosync -r "wlan.*=.*,(PC180049.ORF),(\d+),(\d+),(\d+),(\d+)\";" -o .
photosync -r "wlan.*=.*,(PC180049\.ORF),(\d+),(\d+),(\d+),(\d+)\";" -o .
photosync -r "wlan.*=\"\/DCIM.*,(\\w+\\.ORF),(\d+),(\d+),(\d+),(\d+)\";" -o .
photosync -r "wlan.*=.*,(.*ORF),(\d+),(\d+),(\d+),(\d+).*" -o .
```

Also maybe adding a command line parameter to only copy RAW files could
be useful, or some regex example in the README?

Cheers

**Comment by mauriciojost** [comment]
Hi @XioNoX , 

Thanks for opening this issue! And merry Xmas! :) 

I believe your comment is worth the effort, so I am adding right now a new feature to photosync based on your request. 
The new feature will be based in glob's and not in regexs. Why? I don't expect photosync users to be familiar with regex's. The feature will allow to specify a set of globs to specify which files are to be synchronized, in your case you will be able to specify `*.orf,*.ORF` to match the raw files you want. 

Does this sound good for you?

Mauricio

**Comment by mauriciojost** [comment]
Hi @XioNoX, 

This feature has been implemented. It's available in v0.10, could you please give it a try and let me know if you find any issue?

Mauricio

**Comment by XioNoX** [comment]
I (very) briefly tested v0.11 and it does the job, thanks.
`photosync -o . -P *.ORF`
Only transfered *.ORF files.
